pkgs/servers/tvheadend/default.nix

```diff
@@ -1,82 +1,99 @@
 { lib, stdenv, fetchFromGitHub, cmake, makeWrapper, pkg-config
-, avahi, dbus, gettext, git, gnutar, gzip, bzip2, ffmpeg_4, libiconv, openssl, python2
-, v4l-utils, which, zlib }:
+, avahi, dbus, gettext, git, gnutar, gzip, bzip2, ffmpeg_4, libdvbcsa, libiconv, libopus, libvpx, openssl, python2
+, v4l-utils, which, x264, x265, zlib }:
 
 let
-  version = "4.2.8";
+  versions = {
+    tvheadend = {
+      version = "4.2.8";
+      rev = "v4.2.8";
+      sha256 = "1xq059r2bplaa0nd0wkhw80jfwd962x0h5hgd7fz2yp6largw34m";
+    };
 
-  dtv-scan-tables = stdenv.mkDerivation {
-    pname = "dtv-scan-tables";
-    version = "2020-05-18";
-    src = fetchFromGitHub {
-      owner = "tvheadend";
-      repo = "dtv-scan-tables";
-      rev = "e3138a506a064f6dfd0639d69f383e8e576609da";
-      sha256 = "19ac9ds3rfc2xrqcywsbd1iwcpv7vmql7gp01iikxkzcgm2g2b6w";
+    tvheadend-latest = {
+      version = "master-9a51cea";
```

Comment:
This format should be unstable-XXXX-XX-XX where XXXX-XXXX is the date of the commit.